libs/utils/src/http/error.rs

```diff
@@ -5,8 +5,8 @@ use thiserror::Error;
 
 #[derive(Debug, Error)]
 pub enum ApiError {
-    #[error("Bad request: {0}")]
-    BadRequest(String),
+    #[error("Bad request: {0:?}")]
```

Comment:
Just noticed - this should be `{0:#?}` instead of `{:?}` so that it doesn't have a bunch of newlines. I'll hold off on changing that for now; want to get general feedback first.